mkt/ratings/serializers.py

```diff
@@ -82,6 +90,11 @@ def validate(self, attrs):
             user = self.request.user
             attrs['user'] = user
             attrs['ip_address'] = self.request.META.get('REMOTE_ADDR', '')
+            guessed_lang = guess_language(attrs['body'])
```

Comment:
In general I prefer to avoid surprises or code that requires extra thinking to read.